<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="patient">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditname">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="days">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Days In Hospital</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdays">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="surgery">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Surgery</string>
    </property>
   </widget>
   <widget class="QLabel" name="med">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Medication </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSurgery">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Back</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shoulder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Leg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Knee</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Foot</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboMeds">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Antibiotic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Anti-Nausea</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Anti-inflammatory</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Light Pain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Strong Pain</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addPatient">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewSummary">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View Summary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAndExit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to File</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableSummary">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>201</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total Charges</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
